packages/db/drizzle.ts

```diff
@@ -69,3 +69,4 @@ export const db = initDb();
 export const typedDb: UniversalDb = db;
 
 export { isLocal } from "./utils";
+export * from "./audit-utils";
```

Comment:
We agreed to avoid barrel files and use deep imports instead.

Related 1/: Should isLocal be exported at all? Who uses it?
Related 2/: Suggest to update packages.json to use ESM syntax instead of old CommonJs syntax. Meaning use "exports" instead of "main / types". Have a look at packages/ui for reference (although it doesn't export types).